src/script/interpreter.cpp

```diff
@@ -263,11 +263,11 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
     set_error(serror, SCRIPT_ERR_UNKNOWN_ERROR);
     if (script.size() > MAX_SCRIPT_SIZE)
         return set_error(serror, SCRIPT_ERR_SCRIPT_SIZE);
-    int nOpCount = 0;
     bool fRequireMinimal = (flags & SCRIPT_VERIFY_MINIMALDATA) != 0;
```

Comment:
Yes, there are more vars that could be moved into the tighter `try` scope. I'll take all `interpreter.cpp` changes in a separate PR since they'll need a more thorough review. I've now removed the `nOpCount` move from this PR.